README.md

```diff
@@ -11,6 +11,33 @@ If you do not need to create a new datapath and just want to get started using
 CCP, please see our [guide](https://ccp-project.github.io/ccp-guide).
 
 
+## Build
+
+The simplest way to build is to simply run `make`. By default, we build both a shared and static version of the library.
+
+### cmake
+
+If you want to use `cmake`, run `cmake <path_to_libccp>` from within a separate build directory, then run `make` 
```

Comment:
Will running `cmake` overwrite the Makefile that is there currently?

Reply:
It creates (and thus overwrites any existing) Makefile in your working directory when you run cmake. So if they follow the instructions there and run cmake from a different directory, then no it won't. If they run it from within libccp root then it will